lib/linguist/languages.yml

```diff
@@ -4056,6 +4078,7 @@ Scheme:
   color: "#1e4aec"
   extensions:
   - ".scm"
+  - ".sch"
```

Comment:
I've spent the entire day sorting files into folders, I'm getting jittery and restless. 😆